bakery/src/scripts/jsonify_single.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+exec 2> >(tee jsonified-book/stderr >&2)
```

Comment:
jsonified-book should probably not be jsonified-book (should probably be some param)